src/schema/sale/index.js

```diff
@@ -96,19 +95,22 @@ const SaleType = new GraphQLObjectType({
               "List of artwork IDs to exclude from the response (irrespective of size)",
           },
         },
-        resolve: ({ id }, options) => {
+        resolve: (
+          { id },
+          options,
+          request,
+          { rootVaue: { saleArtworksLoader } }
+        ) => {
           const invert = saleArtworks => map(saleArtworks, "artwork")
 
+          const opts = clone(options)
           if (options.all) {
-            return gravity
-              .all(`sale/${id}/sale_artworks`, options)
-              .then(invert)
-              .then(exclude(options.exclude, "id"))
+            opts.size = 99
```

Comment:
Here's the 99 size for the sale artworks `Other works in this sale` artwork page below-the-fold section.

Reply:
Same, why can’t this paginate instead?